### PR TITLE
script: retry aura swap and clean up $bal balance

### PR DESCRIPTION
**Pull request by gosuto-inzasheru**
relates to #750 not being fully complete yet.

### CODE REVIEW

File: scripts/issue/750/retry_aura_swap.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    )
+
+    bal_to_deposit = bal.balanceOf(vault)
+    wrapper_aurabal_out = wrapper.getMinOut(bal_to_deposit, 9950)
+    bal.approve(wrapper, bal_to_deposit)
+    wrapper.deposit(bal_to_deposit, wrapper_aurabal_out, True, ZERO_ADDRESS)
+
+    # might as well stake all aurabal
+    rewards_balance_before = aurabal_rewards.balanceOf(vault)
+    aurabal_rewards.stakeAll()
```
Comment (petrovska-petro) [inline scripts/issue/750/retry_aura_swap.py:32]: missing approval here
```suggestion
    aurabal.approve(aurabal_rewards, aurabal.balanceOf(vault))
    aurabal_rewards.stakeAll()
```

**Review by petrovska-petro — APPROVED** [review]
nice adding the opp for processing the bal dust and depositing the aurabal to keep rolling the yield ✅ 

note:  it is processing for usdc slightly more aura than what was opened as an order, but amount is minimal and will not leave any dusty aura in the msig

**Comment by gosuto-inzasheru** [comment]
yea im not sure where that aura dust is coming from, but might as well sell it.

**Review by sajanrajdev — APPROVED** [review]
Script properly submits a market sell order for all the AURA on the vault for USDC, deposits the BAL dust for more auraBAL and stakes all of the auraBAL in the vault.
Snapshot:
```
snapshot result for 0xD0A7A8B98957b9CD3cFB9c0425AbE44551158e9e:
                   balance_before         balance_after              balance_delta
symbol
BAL         83.417734696537120118  0.000000000000000000     -83.417734696537120118
auraBAL  1,823.157428422363496826  0.000000000000000000  -1,823.157428422363496826
```

Additionally, it adds True USD to the bribe token claimable list.

LGTM

**Pull request merged**